features/instrumentation.feature

```diff
@@ -89,99 +89,158 @@ Feature: Automatic creation of spans
+      | bugsnag.span.category             | stringValue | app_start            |
+      | bugsnag.app_start.type            | stringValue | cold                 |
+      | bugsnag.app_start.first_view_name | stringValue | SplashScreenActivity |
+
+    * a span named "[AppStartPhase/Framework]" contains the attributes:
+      | attribute             | type        | value           |
+      | bugsnag.span.category | stringValue | app_start_phase |
+      | bugsnag.phase         | stringValue | FrameworkLoad   |
+
+    * a span named "[ViewLoad/Activity]MainActivity" contains the attributes:
+      | attribute             | type        | value        |
+      | bugsnag.span.category | stringValue | view_load    |
+      | bugsnag.view.type     | stringValue | activity     |
+      | bugsnag.view.name     | stringValue | MainActivity |
+
+    * a span named "[ViewLoadPhase/ActivityCreate]MainActivity" contains the attributes:
+      | attribute             | type        | value           |
+      | bugsnag.span.category | stringValue | view_load_phase |
+      | bugsnag.phase         | stringValue | ActivityCreate  |
+      | bugsnag.view.name     | stringValue | MainActivity    |
+
+    * a span named "[ViewLoadPhase/ActivityStart]MainActivity" contains the attributes:
+      | attribute             | type        | value           |
+      | bugsnag.span.category | stringValue | view_load_phase |
+      | bugsnag.phase         | stringValue | ActivityStart   |
+      | bugsnag.view.name     | stringValue | MainActivity    |
+
+    * a span named "[ViewLoadPhase/ActivityResume]MainActivity" contains the attributes:
+      | attribute             | type        | value           |
+      | bugsnag.span.category | stringValue | view_load_phase |
+      | bugsnag.phase         | stringValue | ActivityResume  |
+      | bugsnag.view.name     | stringValue | MainActivity    |
+    * a span named "[ViewLoadPhase/ActivityCreate]MainActivity" contains the attributes:
+      | attribute             | type        | value           |
+      | bugsnag.span.category | stringValue | view_load_phase |
+      | bugsnag.phase         | stringValue | ActivityCreate  |
+      | bugsnag.view.name     | stringValue | MainActivity    |
+
+    * a span named "[ViewLoadPhase/ActivityStart]MainActivity" contains the attributes:
+      | attribute             | type        | value           |
+      | bugsnag.span.category | stringValue | view_load_phase |
+      | bugsnag.phase         | stringValue | ActivityStart   |
+      | bugsnag.view.name     | stringValue | MainActivity    |
+
+    * a span named "[ViewLoadPhase/ActivityResume]MainActivity" contains the attributes:
+      | attribute             | type        | value           |
+      | bugsnag.span.category | stringValue | view_load_phase |
+      | bugsnag.phase         | stringValue | ActivityResume  |
+      | bugsnag.view.name     | stringValue | MainActivity    |
```

Comment:
If I'm reading this right, these are two sets of the same 3 spans?  Could they potentially match on just three unique spans and there be another 3 spans of junk present?  Did you intend for one half of them to cover the new activity?

Reply:
Oops, nicely spotted. That second set should be the new SplashScreenActivity - will fix.